core/arch/arm/plat-stm/link.mk

```diff
@@ -1,69 +1 @@
-link-out-dir = $(out-dir)/core
-
-link-script = $(platform-dir)/kern.ld.S
-link-script-pp = $(link-out-dir)/kern.ld
-link-script-dep = $(link-out-dir)/.kern.ld.d
-
-AWK	 = awk
-
-
-link-ldflags  = $(LDFLAGS)
-link-ldflags += -T $(link-script-pp) -Map=$(link-out-dir)/tee.map
-link-ldflags += --sort-section=alignment
-link-ldflags += --fatal-warnings
-link-ldflags += --print-gc-sections
-
-link-ldadd  = $(LDADD)
-link-ldadd += $(addprefix -L,$(libdirs))
-link-ldadd += $(addprefix -l,$(libnames))
-ldargs-tee.elf := $(link-ldflags) $(objs) $(link-ldadd) $(libgcccore)
-
-link-script-cppflags := -DASM=1 \
-	$(filter-out $(CPPFLAGS_REMOVE) $(cppflags-remove), \
-		$(nostdinccore) $(CPPFLAGS) \
-		$(addprefix -I,$(incdirscore) $(link-out-dir)) \
-		$(cppflagscore))
-
--include $(link-script-dep)
-
-link-script-extra-deps += $(conf-file)
-cleanfiles += $(link-script-pp) $(link-script-dep)
-$(link-script-pp): $(link-script) $(link-script-extra-deps)
-	@$(cmd-echo-silent) '  CPP     $@'
-	@mkdir -p $(dir $@)
-	$(q)$(CPPcore) -Wp,-P,-MT,$@,-MD,$(link-script-dep) \
-		$(link-script-cppflags) $< | \
-		sed -e "s/%in_TEE_SCATTER_START%/$(TEE_SCATTER_START)/g" > $@
```

Comment:
Then you can remove the definition of `TEE_SCATTER_START` in core/arch/arm/plat-stm/system_config.mk (also, `CFG_LINUX_LOAD_ADDR` is not used anymore).